source/scripts/app.js

```diff
@@ -647,13 +649,27 @@ async function populateCookbooksPage() {
   let cookbooks = await indexedDb.getAllCookbooks();
 
   // add each cookbook to the page as a new card
-  for (const cookbook of cookbooks) {
+  let defaultCookbook = null;
+
+  for (let i = 0; i < cookbooks.length; i++) {
     let card = document.createElement("cookbook-card");
-    card.cookbook = cookbook;
+    card.cookbook = cookbooks[i];
+
+    let shadow = card.shadowRoot;
+    let title = shadow.querySelector(".title").innerHTML;
+
+    if (title === DEFAULT_COOKBOOK_NAME) {
+      let button = shadow.getElementById("remove");
+      button.remove();
+      defaultCookbook = card;
+      continue;
+    }
+
     bindCookbookCardButtons(card);
 
     cardContainer.appendChild(card);
   }
+  cardContainer.prepend(defaultCookbook);
```

Comment:
Right now the default cookbook buttons are not functional. We can fix this by moving some logic around
```suggestion
    card.cookbook = cookbooks[i];

    let shadow = card.shadowRoot;
    let title = shadow.querySelector(".title").innerHTML;

    if (title === DEFAULT_COOKBOOK_NAME) {
      let button = shadow.getElementById("remove");
      button.remove();
    }

    bindCookbookCardButtons(card);
    
    if (title === DEFAULT_COOKBOOK_NAME) {
      cardContainer.prepend(card);
    } else {
      cardContainer.appendChild(card);
    }
  }
```